sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -43,7 +43,14 @@ private[sql] object PythonSQLUtils {
   def listSQLConfigs(): Array[(String, String, String, String)] = {
     val conf = new SQLConf()
     // Py4J doesn't seem to translate Seq well, so we convert to an Array.
-    conf.getAllDefinedConfs.toArray
+    conf.getAllDefinedConfs.filterNot(p => SQLConf.staticConfKeys.contains(p._1)).toArray
+  }
+
+  def listStaticSQLConfigs(): Array[(String, String, String, String)] = {
+    val conf = new SQLConf()
+    // Force to build static SQL configurations
+    StaticSQLConf.WAREHOUSE_PATH.key -> ()
```

Comment:
what's special about `WAREHOUSE_PATH`?

Reply:
Nothing special, just find something to invoke? Couldn't find another proper way to build static ones to SQLConf, any advice?

Reply:
just write `StaticSQLConf` doesn't work?

Reply:
thanks, it works. there must be some wrong with my environment when I try this way last time.